app/styles/fonts/openshift-logos-icon/demo.html

```diff
@@ -0,0 +1,936 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>OpenShift logos font set</title>
+    <meta name="description" content="An Icon Font Generated By IcoMoon.io">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="demo.css">
+    <link rel="stylesheet" type="text/css" href="../../main.css">
```

Comment:
instead of requiring someone to run the console locally, i'd prefer they be able to just look at this demo file in the repo using rawgit, so this should reference the relative path to the dist/styles/main.css in github